app/components/service/serviceController.js

```diff
@@ -291,15 +295,22 @@ function ($q, $scope, $stateParams, $state, $location, $anchorScroll, ServiceSer
     .then(function success(data) {
       $scope.tasks = data.tasks;
       $scope.nodes = data.nodes;
+
       $scope.secrets = data.secrets.map(function (secret) {
         return new SecretViewModel(secret);
       });
+
+      $timeout(function() {
```

Comment:
the $anchorScroll has to occur after the page has been fully rendered, (otherwise the anchor doesn't exist). This can be achieved using $timeout()